recipes/cylc-uiserver/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cylc-uiserver" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4cc85a6680148cddd8f1fd714f772acb7c52487239b7c2b781bf543a51c099d1
+
+build:
+  number: 0
+  # cylc-uiserver provides JupyterHub extensions and a small Jupyter Application. As JupyterHub
+  # does not run on Windows, we skip the build for that platform here
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
```

Comment:
As this recipe is not ``noarch: python`` you will need to skip those python versions on ``build``

Reply:
```suggestion
    - python
```